source/features/batch-mark-files-as-viewed.tsx

```diff
@@ -26,10 +26,10 @@ function batchToggle(event: delegate.Event<MouseEvent, HTMLFormElement>): void {
 	const previousFileState = isChecked(previousFile);
 	const thisFile = event.delegateTarget.closest<HTMLElement>('.js-file')!;
 	const files = select.all('.js-file');
-	const selectedFiles = files.slice(
+	const selectedFiles = files.slice(...[
 		files.indexOf(previousFile) + 1,
 		files.indexOf(thisFile) + 1
-	);
+	].sort((a, b) => a - b));
```

Comment:
I suggest you use GitHub Desktop or similar to commit only the files and lines that are necessary — and then look at the PR and see if there's any unwanted code. We shouldn't have to comment on this. This file is unrelated